test_libs/gen_helpers/gen_base/gen_suite.py

```diff
@@ -0,0 +1,15 @@
+from eth_utils import (
+    to_dict,
+)
+
+from gen_base.gen_typing import TestSuiteTestCases
+
+
+@to_dict
+def render_suite(*, title: str, summary: str, fork: str, config: str, test_cases: TestSuiteTestCases):
```

Comment:
```suggestion
def render_suite(*, title: str, summary: str, fork: str, config: str, test_cases: Iterable[TestCase]):
```

should suffice and allows users to pass a generator or a list if they want to for some reason. Alternatively, define `TestSuiteTestCases = Iterable[TestCase]` in `gen_typing.py`, but I don't see why.

Reply:
Good suggestion. Went a little overboard with typing there. Also fixed a bit of formatting. See 
25dbfee